tests/ui/autodiff/flag_nott.rs

```diff
@@ -0,0 +1,19 @@
+//@ compile-flags: -Zautodiff=Enable,NoTT
+//@ needs-enzyme
+//@ check-pass
+
+#![feature(autodiff)]
+
+use std::autodiff::autodiff_reverse;
+
+// Test that NoTT flag is accepted and doesn't cause compilation errors
```

Comment:
add a `//@ CHECK-NOT: enzyme_type` (or similar, the syntax might be slightly different).

Reply:
Ah, I see that you already do that in another test, then it's fine.